application/src/main/java/org/thingsboard/server/service/queue/processing/TbRuleEngineProcessingStrategyFactory.java

```diff
@@ -69,6 +76,12 @@ public RetryStrategy(String queueName, boolean retrySuccessful, boolean retryFai
             this.maxRetries = configuration.getRetries();
             this.maxAllowedFailurePercentage = configuration.getFailurePercentage();
             this.pauseBetweenRetries = configuration.getPauseBetweenRetries();
+            this.expPauseBetweenRetries = configuration.isExpPauseBetweenRetries();
+            if (this.expPauseBetweenRetries) {
+                this.expDegreeStep = new AtomicInteger(1);
+                this.maxExpPauseBetweenRetries = configuration.getMaxExpPauseBetweenRetries();
+                this.maxExpDegreeValue = Math.log(maxExpPauseBetweenRetries) / Math.log(pauseBetweenRetries);
```

Comment:
This variable is redundant. You can use expDegreeStep and just compare the `maxExpPauseBetweenRetries` and `new Double(Math.pow(pauseBetweenRetries, expDegreeStep.getAndIncrement())).longValue();`